tools/tertiary-analysis/scanpy/scanpy-run-pca.xml

```diff
@@ -12,21 +12,21 @@ PYTHONIOENCODING=utf-8 scanpy-run-pca.py
     -f '${input_format}'
     -o output.h5
     -F '${output_format}'
-    -n '${n_pc}'
+    -n '${n_pcs}'
     #if $run_mode.chunked
         -c
         --chunk-size '${run_mode.chunk_size}'
     #else
-        #if $run_mode.zero_center
+        #if $run_mode.zero_centre
```

Comment:
Sorry for being inconsistent with spelling flavour myself... Perhaps `zero_centre` -> `zero_center`?

Reply:
yes, scanpy-scripts uses zero-center. I will change this one.